README.md

```diff
@@ -42,7 +42,7 @@ You will also need to install:
       * You also need to install the `Command Line Tools` via Xcode. You can find this under the menu `Xcode -> Preferences -> Downloads`
       * This step will install `gcc` and the related toolchain containing `make`
   * On Windows:
-    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator).
+    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator). Note: although windows-build-tools will install Python, it will not add python to your PATH, so you will have to add it manually.
```

Comment:
I think that should be optional. 

As @felixrieseberg said we didn't want to automatically add python to the`PATH` to prevent messing up users' configuration. But in some cases users don't have python installed and they want `windows-build-tools` to install and add python to their `PATH`. @joshgav Your proposed way looks good, but we need to add command line options to enable it.

This has been sitting in our backlog, but we haven't gotten into implementing it :)